bin/reth/src/dirs.rs

```diff
@@ -55,6 +62,19 @@ impl XdgPath for DbPath {
     }
 }
 
+/// Returns the path to the default JWT secret hex file.
+///
+/// Refer to [dirs_next::data_dir] for cross-platform behavior.
+#[derive(Default, Debug, Clone)]
+#[non_exhaustive]
+pub struct JwtSecretPath;
+
+impl XdgPath for JwtSecretPath {
+    fn resolve() -> Option<PathBuf> {
+        jwt_secret_dir().map(|p| p.join("jwt.hex"))
```

Comment:
is this something standard or did we choose this

Reply:
This is from [specs](https://github.com/ethereum/execution-apis/blob/main/src/engine/authentication.md#key-distribution)